clients/java/signalr/src/test/java/com/microsoft/aspnet/signalr/test/HubConnectionTest.java

```diff
@@ -595,6 +619,38 @@ public void SendWithEightParamsTriggersOnHandler() throws Exception {
         assertEquals("F", value8.get());
     }
 
+    private class Custom {
+        public int number;
+        public String str;
+        public boolean[] bools;
+    }
+
+    @Test
+    public void SendWithCustomObjectTriggersOnHandler() throws Exception {
```

Comment:
Can you just fix all the camel-casing in the tests once and for all? ... it's ... bothering me 😸